src/utils/chart.js

```diff
@@ -1,5 +1,5 @@
 import { _ } from '.';
-import { CHART_SQUARE_TYPES } from '../constants/constants';
+import { CHART_SQUARE_TYPES } from 'src/constants';
```

Comment:
looking at line one, makes me think './_' makes more sense. doing '.' imports from the index and this could cause circular dependencies later on